includes/ProductSync/ProductExcludedException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\ProductSync;
+
+use Exception;
+
+/**
+ * Class ProductExcludedException
+ *
+ * Exception for when a product is excluded from Facebook product sync.
+ */
+class ProductExcludedException extends Exception {}
```

Comment:
Might be clearer to name this `ProductSyncExcludeException`.

Reply:
Similarly, I see the exception message strings don't mention sync - I think it's clearer to mention sync everywhere, so it's clear what this is part of.

Reply:
Can do. These classes are in the `ProductSync` namespace but I'm not against includeing "sync". Actually `ExcludedFromSyncException` could be a helpful name for this class.